package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@protonapp/material-components": "https://component-marketplace-prod.s3.amazonaws.com/@protonapp/material-components/0.8.17.tar",
-    "@protonapp/proton-runner": "^0.2.30",
+    "@protonapp/proton-runner": "^0.4.71",
```

Comment:
That's a pretty significant bump. Are there things you are worried could break?

Reply:
There shouldn't be, I'm not aware of any breaking changes we've made to runner and this simply brings it in line with the rest of our code base. This is also solely developer facing and won't effect makers/end users if we do need to revert for any reason.

At some point we should go through and make sure all dependencies in `mobile-previewer` match the versions in `template-app` so we can be sure that component developer's are testing their native components on an up to date version of Adalo

Reply:
Sounds good. Thanks

🛳  it